tools/fetch_fasta_from_ncbi/retrieve_fasta_from_NCBI.py

```diff
@@ -56,6 +56,19 @@ def __init__(self, options, logger):
         self.count = 0
         self.webenv = ""
         self.query_key = ""
+        self.datetype = options.datetype
+        if options.reldate:
+            self.reldate = options.reldate
+        else:
```

Comment:
else is in case the option isn't used there is no need to check it afterwards.
In this case the url will only state &reldate=&nextoption.
If I don't use it I have to check if the option exists with an if anyway.